llvm/unittests/Analysis/VectorFunctionABITest.cpp

```diff
@@ -44,10 +46,11 @@ class VFABIParserTest : public ::testing::Test {
   FunctionCallee F;
 
 protected:
-  // Referencies to the parser output field.
+  // References to the parser output field.
   ElementCount &VF = Info.Shape.VF;
   VFISAKind &ISA = Info.ISA;
   SmallVector<VFParameter, 8> &Parameters = Info.Shape.Parameters;
```

Comment:
wouldn't be more clear to call VecFuncParamaters? to avoid confusion in the checks like:

  EXPECT_EQ(Parameters.size(), (unsigned)5);
  EXPECT_TRUE(matchScalarParametersNum());

where it is not clear why we check it twice.
WDYT?